core/src/main/java/org/bitcoinj/core/ECKey.java

```diff
@@ -23,7 +23,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
-import com.google.common.base.Objects;
+import java.util.Objects;
```

Comment:
Import order isn't right here (multiple occurences).

Reply:
Good catch. I'll fix it